apps/consulting/posts/openai-pseudocode.md

```diff
@@ -0,0 +1,81 @@
+---
+title: 'Collaborative code generation with LLMs'
+published: March 27, 2023
+author: chris-ostrouchov
+description: 'As an experiment we developed a python package pseudocode which much like its name implies allows developers to annotate their code with pseudocode and produce testable code generated by LLMs without the user writing any code.'
+category: ["ML", "LLM", "python", "openai"]
+featuredImage:
+  src: '/posts/openai-pseudocode/pseudocode-screenshot.png'
+  alt: 'A screenshot of the pseudocode program running'
+hero:
+  imageSrc: /posts/hero-paris.webp
```

Comment:
this should be changed. I don't think a paris viz makes sense as a hero image for this.

Reply:
Be aware that the way the header is currently designed, it's necessary to have a dark hero.

Otherwise, the header navigation will be a light gray against a light background, making it very hard to see.